scripts/python/tao_build.py

```diff
@@ -412,51 +302,43 @@ def bazel_build(target, flag=""):
 
 @time_stage()
 def build_mlir_ral(root, args):
-    BAZEL_BUILD_CMD = "bazel build --experimental_multi_threaded_digest --define framework_shared_object=false"
+    configs = ['--config=cxx11abi_{}'.format(int(args.ral_cxx11_abi))]
     if not args.cpu_only:
         if args.dcu:
-            BAZEL_BUILD_CMD = BAZEL_BUILD_CMD + " --config=dcu"
+            configs.append('--config=disc_dcu')
         else:
-            BAZEL_BUILD_CMD = BAZEL_BUILD_CMD + " --config=cuda"
+            configs.append('--config=disc_cuda')
     else:
-        BAZEL_BUILD_CMD = BAZEL_BUILD_CMD + " --config=release_cpu_linux"
+        configs.append('--config=disc_cpu')
 
-    BAZEL_BUILD_CMD += ci_build_flag()
+    if args.enable_blaze_opt:
+        configs.append('--config=disc_blaze')
+
+    if args.enable_mkldnn:
+        configs.append('--config=disc_mkldnn')
+
+    if running_on_ci():
+        configs.append('--config=ci_build')
+
+    BAZEL_BUILD_CMD = "bazel build --experimental_multi_threaded_digest --define framework_shared_object=false"
```

Comment:
could this be replaced with `--config=disc`?  or just simply remove these options since above config will also apply these?

Reply:
I deleted the parameters here because they are redundant. But, I would like to make disc configurable but not hard coding here.